src/aura/disbursementsCreate/disbursementsCreate.cmp

```diff
@@ -12,22 +12,22 @@
     <div class="wrap">
 
         <!-- Header -->
-        <h2 class="slds-text-heading_medium slds-hyphenate slds-m-bottom--large">Schedule Disbursements</h2>
+        <h2 class="slds-text-heading_medium slds-hyphenate slds-m-bottom--large">{!$Label.c.disbursementsCreate_ScheduleDisbursements}</h2>
 
         <div class="slds-section slds-is-open">
 
             <!-- Funding Request Data -->
             <h3 class="slds-section__title slds-theme_shade">
-                <span class="slds-truncate slds-p-horizontal_small" title="Basics">Funding Request Information</span>
+                <span class="slds-truncate slds-p-horizontal_small" title="Basics">{!$Label.c.disbursementsCreate_FundingRequestInformation}</span>
```

Comment:
Don't we need translation for the `title` attribute since it shows as tooltip?

Reply:
great catch